app/javascript/components/karoake.js

```diff
@@ -0,0 +1,85 @@
+import { pause } from "./composition_player";
+
+// import * as flacFactory from 'libflacjs';
+// const Flac = flacFactory();
+
+// import { Encoder } from 'libflacjs/lib/encoder';
+
+const karoakePlayer = () => {
```

Comment:
Karaoke is spelled incorrectly!! lol 
Filename is also spelled incorrectly! - karoake.js